filament/src/MorphTargetBuffer.cpp

```diff
@@ -106,7 +106,7 @@ FMorphTargetBuffer::FMorphTargetBuffer(FEngine& engine, const Builder& builder)
           mCount(builder->mCount) {
     FEngine::DriverApi& driver = engine.getDriverApi();
     mSbHandle = driver.createSamplerGroup(PerRenderPrimitiveMorphingSib::SAMPLER_COUNT);
-    mPbHandle = driver.createTexture(SamplerType::SAMPLER_2D_ARRAY, 1, TextureFormat::RGB32F, 1,
+    mPbHandle = driver.createTexture(SamplerType::SAMPLER_2D_ARRAY, 1, TextureFormat::RGBA16F, 1,
```

Comment:
I'm not understanding this change. I don't think positions can be stored as fp16, it's not enough precision. It's okay for normals usually, though.

Reply:
Ok I will revert it. I agree because fp32 is used for position at the vertex input. Could you talk about this with @romainguy? He want to this.

Reply:
I think we could make it an option, but if it's not an option it should be "correct first", so it has to be RGBA32F for positions. Sorry for the back-and-forth.

Reply:
That's fine. :) I also think position's format should be RGBA32F.

Reply:
It's reverted. Thanks.